help.php

```diff
@@ -0,0 +1,124 @@
+    <div class="col-md-12">
+    <h2>Main page</h2>
+    <p>On the main page, various statistics of pi-hole are shown to the user:</p>
+    <ul>
+        <li>Summary: A summary of statistics showing how many out of how many total DNS queries have been blocked today, how that translates into a percentage and how many domains are on the blacklist. This graph is updated every 10 seconds. Changes are highlighted.</li>
+        <li>Queries over time: Diagram showing DNS queries (total and blocked) over 10 minute time intervals. More information can be acquired by hovering over the lines.</li>
+        <li>Query Types: Shows which types of query have been processed:
+            <ul>
+                <li>A: IPv4 address lookup (most commonly used to map hostnames to an IP address of the host)</li>
+                <li>AAAA: IPv6 address lookup (most commonly used to map hostnames to an IP address of the host)</li>
+                <li>PTR: most common use is for implementing reverse DNS lookups</li>
+                <li>SRV: Service locator (often used by XMPP, SIP, and LDAP)</li>
+                <li>and others</li>
+            </ul>
+        </li>
+        <li>Query Types: Shows to which upstream DNS the permitted requests have been forwarded to.</li>
+        <li>Top Domains: Ranking of requested sites by number of DNS lookups.</li>
+        <li>Top Advertisers: Ranking of requested advertisements by number of DNS lookups.</li>
+        <li>Top Clients: Ranking of total DNS requests separated by clients on the local network.</li>
+    </ul>
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-12">
+    <h2>Query Log</h2>
+    <p>Shows the recent queries after parsing the pi-hole log files. It is possible to search through the whole list by using the "Search" input field. If the status is reported as "OK", then the DNS request has been permitted. Otherwise ("Pi-holed") it has been blocked. By clicking on the buttons under "Action" the corresponding domains can quickly be added to the white-/blacklist. The status of this action will be reported on this page.</p>
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-12">
+    <h2>White- / Blacklist</h2>
+    <p>Add or remove domains (or subdomains) from the white-/blacklist. If a domain is added to e.g. the whitelist, any possible entry of the same domain will be automatically removed from the blacklist and vice versa. Adding wildcards using the web UI is currently <em>not</em> supported.</p>
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-12">
+    <h2>Update Lists</h2>
+    <p>Runs the command <pre>sudo pihole -g</pre> and prints the result transparently to the web UI. The gravity.sh script will update the list of ad-serving domains</p>
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-12">
+    <h2>Disable / Enable</h2>
+    Disables resp. enables Pi-Hole DNS Blocking completely. The change will be reflected by a changed status (top left)
```

Comment:
"Disables/enables Pi-Hole blocking completely. You may have to wait a few minutes for the changes to reach all of your devices. The change will be reflected by a changed status (top left)"